cdap-data-fabric/src/test/java/co/cask/cdap/data2/metadata/lineage/SqlLineageTableTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.data2.metadata.lineage;
+
+import co.cask.cdap.data2.sql.PostgresSqlStructuredTableAdmin;
+import co.cask.cdap.data2.sql.SqlStructuredTableRegistry;
+import co.cask.cdap.data2.sql.SqlTransactionRunner;
+import co.cask.cdap.spi.data.StructuredTableAdmin;
+import co.cask.cdap.spi.data.TableAlreadyExistsException;
+import co.cask.cdap.spi.data.table.StructuredTableRegistry;
+import co.cask.cdap.store.StoreDefinition;
+import com.opentable.db.postgres.embedded.EmbeddedPostgres;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+import java.io.IOException;
+import javax.sql.DataSource;
+
+public class SqlLineageTableTest extends LineageTableTest {
+
+  private static EmbeddedPostgres pg;
+
+  @BeforeClass
+  public static void beforeClass() throws IOException, TableAlreadyExistsException {
+    pg = EmbeddedPostgres.start();
```

Comment:
please see here for how to set the temp directory correctly for the embedded postgres: https://github.com/cdapio/cdap/pull/11063/files#diff-144810ba58266142b9892beffc9c38c6R57